README.md

```diff
@@ -13,12 +13,14 @@ Supported shapes:
 
 Note: Bodies for meshes compute the convex hull of those meshes in order to provide the point containment / ray intersection routines.
 
+Note: There was a recent change in point containment computations - now all bodies are expected to also contain their surface points. [The old behavior was inconsistent](https://github.com/ros-planning/geometric_shapes/issues/91). Rounding errors may still yield surprising results, though.
+
 Note: [shape_tools](https://github.com/ros-planning/shape_tools) package was recently merged into this package
 
 ## Build Status
 
 Travis CI: [![Build Status](https://travis-ci.org/ros-planning/geometric_shapes.svg?branch=indigo-devel)](https://travis-ci.org/ros-planning/geometric_shapes)
 
-Devel Job: [![Build Status](http://build.ros.org/buildStatus/icon?job=Jbin_uT64__geometric_shapes__ubuntu_trusty_amd64__binary)](http://build.ros.org/view/Ibin_uT64/job/Jbin_uT64__geometric_shapes__ubuntu_trusty_amd64__binary/)
+Devel Job: [![Build Status](http://build.ros.org/buildStatus/icon?job=Ibin_uT64__geometric_shapes__ubuntu_trusty_amd64__binary)](http://build.ros.org/view/Ibin_uT64/job/Ibin_uT64__geometric_shapes__ubuntu_trusty_amd64__binary/)
```

Comment:
Why did you switched these badges from Jade to Indigo?
They should actually be Melodic...

Reply:
This is targeted at indigo-devel... The badges as were here were broken: `view/Ibin_uT64/job/Jbin_uT64__geometric_shapes` (notice `Ibin` first and `Jbin` second).

Reply:
Sure, I can reorganize. Do you know if force-pushing is okay when the branch is involved in a PR?

Reply:
Yes, force-pushing is frequently used to cleanup PR branches.

Reply:
The breakage risk to Indigo doesn't seem worth it to me. @peci1 could you start working off of the melodic-devel branch instead? are you still on indigo for everything?

Reply:
When I started this PR, we were fully on indigo, now we're transitioning to melodic. Sure, I can rebase this PR on top of melodic devel. Kinetic remains a question, though...

Reply:
But now I'm a bit confused as I see you also approved this PR...

Reply:
@davetcoleman ?

Reply:
@davetcoleman See #97 - I've retargeted this PR to melodic.